azuread/resource_application_test.go

```diff
@@ -414,6 +444,16 @@ resource "azuread_application" "test" {
 `, id)
 }
 
+func testAccADApplication_nativeReplyUrls(id string) string {
+	return fmt.Sprintf(`
+resource "azuread_application" "test" {
+  name       = "acctest%s"
+	type       = "native"
```

Comment:
could we use spaces instead of tabs here?

Reply:
gets me every time, vscode swaps out spaces for tabs in .go files by default